### ISSUE TITLE
Add links to interoperable calendar format files for all workshop sessions

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Maps4HTML-Workshop-2020#78]
As discussed, but not logged here, we should make a download link available for calendar entries for the workshop sessions, which will help ensure people don't forget about it.

**Comment by AmeliaBR** [comment]
iCal is a text-based format, so we should be able to generate it from our YAML data structure using Jekyll. @nchan0154 do you want to take a look at this?
https://tools.ietf.org/html/rfc5545

**Comment by nchan0154** [comment]
I have looked into generating .ics files with Jekyll. While it is definitely possible (here is an [example I found](https://github.com/crtr0/seattlehacks/blob/gh-pages/events.ics)), it requires a plugin (either prewritten or a custom one), and custom plugins are not allowed by Github Pages.  An alternative solution is to [generate the files on the client ](https://github.com/nwcell/ics.js) when a button is pressed. If you think this works ok (ie. If the schedule changes,  there won't be a way to propagate changes to the attendees calendars), I can set this up for us, but it may be worth looking into other options if the schedule will change a lot.
